installers/windows/Monero.iss

```diff
@@ -39,7 +39,7 @@ Name: "en"; MessagesFile: "compiler:Default.isl"
 ; .exe/.dll file possibly with version info).
 ;
 ; This is far more robust than relying on version info or on file dates (flag "comparetimestamp").
-; As of version 0.11.1.0, the Monero .exe files do not carry version info anyway in their .exe headers.
+; As of version 0.12.0.0, the Monero .exe files do not carry version info anyway in their .exe headers.

```

Comment:
Is that right ?

Reply:
Yes. To be sure I just checked the latest buildbot build: No version number in the sense of Windows in there, nor any application name or copyright that you could put into the .exe header as well.

I don't know how one would have to modify the make file to make MSYS2 `ld` put the current version number into the Windows .exe header.